src/tests/functional/plugin/shared/include/behavior/ov_infer_request/inference.hpp

```diff
@@ -0,0 +1,93 @@
+// Copyright (C) 2018-2021 Intel Corporation
```

Comment:
may be we can give more proper name for file like `roi_tensor_inference.hpp`?

Reply:
My idea was to create generic class for inference test of various infer_requests and tensors. ROI-tensor test is just one usage of this.
Generally, to this test user needs to pass input shape of function, input tensor (can have different shape, such as ROI), and expected data after inference.
To make it more generic, it would be possibly good to allow customize 'model' and number of inputs/outputs, but it becomes too complicated and not necessary, at least for now